.rubocop.yml

```diff
@@ -3,6 +3,7 @@ inherit_from: .rubocop_todo.yml
 
 # Apply rule to all cops
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
what happen if you set the ruby version and do not disable the rules? (leave this change and remove the changes of `rubocop_todo`)

Reply:
`568 files inspected, 614 offenses detected`

Reply:
I had to push `TargetRubyVersion` all the way down to 2.2 in order to get a clean pass.